onnx2kerastl/operation_layers.py

```diff
@@ -889,10 +890,19 @@ def get_empty_array(x, dtype=new_dtype, keras_name=keras_name):
 
 
 def convert_einsum(node, params, layers, lambda_func, node_name, keras_name):
-    # input_0 = layers[node.input[0]]
-    # input_1 = layers[node.input[1]]
-    # equation = params['equation'].decode('utf-8')
-    # layers[node_name] = tf.einsum(equation, *[input_0, input_1], name=keras_name)
-    input_0 = tf_expand_dims(layers[node.input[0]], axis=2, tf_name=f"{params['cleaned_name']}_einsum_0")
-    input_1 = tf_expand_dims(layers[node.input[1]], axis=0, tf_name=f"{params['cleaned_name']}_einsum_1")
-    layers[node_name] = tf_multiply(input_0, input_1, tf_name=f"{params['cleaned_name']}_einsum_mult")
+    input_0 = layers[node.input[0]]
+    input_1 = layers[node.input[1]]
+    equation = params['equation'].decode('utf-8')
+
+    is_input_0_constant = isinstance(input_0, tf.Tensor)
+    is_input_1_constant = isinstance(input_1, tf.Tensor)
```

Comment:
@tip3x 
For both tests above, this could be a `numpy` as well, depending on the specifics of the model. Can you add a test for that?